applications/luci-app-haproxy-tcp/luasrc/model/cbi/haproxy.lua

```diff
@@ -8,19 +8,17 @@ function sync_value_to_file(value, file) --用来写入文件的函数，目前
 		nixio.fs.writefile(file, value)
 	end	
 end
-local state_msg = ""
-local haproxy_on = (luci.sys.call("pidof haproxy > /dev/null") == 0)
+
 local router_ip = luci.sys.exec("uci get network.lan.ipaddr")
-if haproxy_on then	
-	state_msg = "<b><font color=\"green\">" .. translate("Running") .. "</font></b>"
-else
-	state_msg = "<b><font color=\"red\">" .. translate("Not running") .. "</font></b>"
-end
-m=Map("haproxy",translate("HAProxy"),translate("HAProxy能够检测Shadowsocks服务器的连通情况，从而实现负载均衡和高可用的功能，支持主备用服务器宕机自动切换，并且可以设置多个主服务器用于分流，规定每个分流节点的流量比例等。前提条件是你的所有Shadowsocks服务器的【加密方式】和【密码】一致。<br><br>使用方法：配置好你的Shadowsocks服务器ip地址和端口，然后开启Shadowsocks服务，将服务器地址填写为【127.0.0.1】，端口【2222】，其他参数和之前一样即可，你可以通过访问【路由器的IP:1111/haproxy】输入用户名admin，密码root来观察各节点健康状况，红色为宕机，绿色正常,使用说明请<a href='http://www.right.com.cn/forum/thread-198649-1-1.html'>点击这里</a>") .. "<br><br>后台监控页面：<a href='http://" .. router_ip .. ":1111/haproxy'>" .. router_ip .. ":1111/haproxy</a>  用户名admin，密码root" .. "<br><br>状态 - " .. state_msg)
-s=m:section(TypedSection,"arguments","")
-	s.addremove=false
-	s.anonymous=true
-	view_enable = s:option(Flag,"enabled",translate("Enable"))
+
+m = Map("haproxy", translate("HAProxy"), translate("HAProxy能够检测Shadowsocks服务器的连通情况，从而实现负载均衡和高可用的功能，支持主备用服务器宕机自动切换，并且可以设置多个主服务器用于分流，规定每个分流节点的流量比例等。前提条件是你的所有Shadowsocks服务器的【加密方式】和【密码】一致。<br><br>使用方法：配置好你的Shadowsocks服务器ip地址和端口，然后开启Shadowsocks服务，将服务器地址填写为【127.0.0.1】，端口【2222】，其他参数和之前一样即可，你可以通过访问【路由器的IP:1111/haproxy】输入用户名admin，密码root来观察各节点健康状况，红色为宕机，绿色正常,使用说明请<a href='http://www.right.com.cn/forum/thread-198649-1-1.html'>点击这里</a>") .. "<br><br>后台监控页面：<a href='http://" .. router_ip .. ":1111/haproxy'>" .. router_ip .. ":1111/haproxy</a>  用户名admin，密码root")
+
+a:section(SimpleSection).template  = "haproxy/haproxy_status"
+
```

Comment:
So where `a` got defined? Name status page to `sth/status` instead of `sth/sth_status`. The latter one is just repeating.